docs/resources/check_group.md

```diff
@@ -54,9 +54,19 @@ resource "checkly_check_group" "test_group1" {
       password = "pass"
     }
   }
-  environment_variables = {
```

Comment:
This method of declaring environment variables was deprecated and produces a warning.